lib/twilio-ruby/util.rb

```diff
@@ -1,10 +1,12 @@
 module Twilio
   module Util
     def url_encode(hash)
+      warn "'Twilio::Util::url_encode has been deprecated."
```

Comment:
double quoted? `"` and `'` in `"'Twilio::Util`?

Reply:
To provide clarity in CLI output.